src/Sets/SimpleSparsePolynomialZonotope.jl

```diff
@@ -152,3 +152,45 @@ The set resulting from applying the linear map `M` to `P`.
 function linear_map(M::AbstractMatrix, P::SSPZ)
     return SimpleSparsePolynomialZonotope(M * center(P), M * genmat(P), expmat(P))
 end
+
+"""
+    quadratic_map(Q::Vector{MT}, S::SimpleSparsePolynomialZonotope) where {N, MT<:AbstractMatrix{N}}
+
+Return an overapproximation of the quadratic map of the given zonotope.
+
+### Input
+
+- `S` -- simple sparse polynomial zonotope
+- `Q` -- array of square matrices
```

Comment:
```suggestion
- `Q` -- vector of square matrices
- `S` -- simple sparse polynomial zonotope
```
to match the order of the arguments